samples/boards/sensortile_box_pro/ble-fw-upgrade/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _sensortile_box_pro_sample_ble_fwupg:
+
+ST SensorTile.box Pro BLE Firwmare Upgrade
```

Comment:
I would recommend using `.. zephyr:code-sample::` directive, see other samples using it for ... well... examples :D

Reply:
seems cool!